src/App.js

```diff
@@ -13,25 +13,39 @@ import NotFound from "./pages/404/404";
 import Footer from "./components/Footer/Footer";
 import Design from "./pages/Design/Design";
 import { Helmet } from "react-helmet";
-import ImageTemplate from "./pages/Design/imageTemplate";
```

Comment:
Why did you remove the routes to the images?

Reply:
Oh sorry, i guess because i created different branches for two issues, this one didnt have the changes.